src/deadline/client/cli/_groups/bundle_group.py

```diff
@@ -81,6 +83,85 @@ def validate_parameters(ctx, param, value):
     return parameters_split
 
 
+def _validate_submitter_info(ctx, param, value):
+    """
+    Validate provided --submitter-info value and convert to SubmitterInfo object.
+
+    Supports two input formats (mutually exclusive):
+    - Multiple key=value pairs: --submitter-info submitter_name=MyApp --submitter-info host_application_name=Maya
+    - Single inline JSON blurb: --submitter-info '{"submitter_name": "MyApp", "additional_info": {"custom": "data"}}'
+
+    All keys must be valid SubmitterInfo fields. Unknown keys will raise an error.
+    """
+    if not value:
+        return None
+
+    # Get valid field names from SubmitterInfo dataclass
+    valid_fields = {field.name for field in fields(SubmitterInfo)}
+
+    # Check if using JSON format (only one value allowed and it must start with {)
+    if len(value) == 1 and value[0].startswith("{"):
```

Comment:
Might be simpler to commit to one data format instead of accepting JSON and key-value pairs.